dev/com.ibm.ws.security.oidc.client_fat.2/fat/src/com/ibm/ws/security/openidconnect/client/fat/IBM/OidcPropagationConsumeUserinfoTests.java

```diff
@@ -0,0 +1,1213 @@
+            expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS, "Client messages.log should contain a message indicating that the auth endpoint is not set.", MessageConstants.CWWKS1534E_MISSING_AUTH_ENDPOINT);
+            if (isJWS) {
+                expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS, "Client messages.log should contain a message indicating that there is a signature mismatch.", MessageConstants.CWWKS1777E_SIG_ALG_MISMATCH);
+            } else {
+                expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS, "Client messages.log should contain a message indicating that there a problem validating the access token.", MessageConstants.CWWKS1740W_RS_REDIRECT_TO_RP);
+                expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.MESSAGES_LOG, Constants.STRING_CONTAINS, "Client messages.log should contain a message indicating that there a problem extracting JWS from JWE.", MessageConstants.CWWKS6056E_ERROR_EXTRACTING_JWS_PAYLOAD_FROM_JWE);
+            }
+        } else {
+            if (expectations == null) {
+                expectations = vData.addSuccessStatusCodes(null);
+            }
+            for (NameValuePair parm : parms) {
+                if (parm.getName().equals("sub") || userinfoWillBeUsed) {
+                    if (parm.getName().equals(JwtConstants.PARAM_KEY_MGMT_ALG) || parm.getName().equals((JwtConstants.PARAM_ENCRYPT_KEY))) {
+                        Log.info(thisClass, "createRSExpectations", "Not adding a check for parm: " + parm + " - attr is in the JWE header and not part of the payload.");
+                    } else {
+                        Log.info(thisClass, "createRSExpectations", "adding extra claim check");
+                        expectations = vData.addExpectation(expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.RESPONSE_FULL, Constants.STRING_MATCHES, "Did not see the claim " + parm.getName() + " in the response.", null, "\"" + parm.getName() + "\":\"" + parm.getValue() + "\"");
+                    }
+                } else {
+                    expectations = vData.addExpectation(expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.RESPONSE_FULL, Constants.STRING_DOES_NOT_MATCH, "Did not see the claim " + parm.getName() + " in the response.", null, "\"" + parm.getName() + "\":\"" + parm.getValue() + "\"");
```

Comment:
Same thing here as in other comment.

Reply:
updated the error message and will deliver with changes to share code between oidc and social